mrr/src/main/java/gov/cdc/dex/mrr/VocabClient.kt

```diff
@@ -38,4 +46,26 @@ class VocabClient {
        // println("VocabKey :${vocabKey}")
         return vocabKey
     }
+
+    fun  setValueSetConcepts(valuesetConcepts: List<ValueSetConcept>,  key: String) {
+
+        var jedis = RedisUtility().redisConnection()
+        if(jedis != null) {
+            println("Cache Response : " + jedis.ping())
+            try {
+                valuesetConcepts?.let {
+                    for (i in valuesetConcepts) {
+                        var vkey = i.conceptCode
+                        jedis.hset(key.toString(), vkey, i.toString())
+                        // println("Valueset in Radis:" + key.toString() + "-" + vkey + "-" + i.toString())
+                    }
+                }
+
+            } catch (e: Exception) {
+                e.printStackTrace()
```

Comment:
again, do not swallow exception  - this is probably why you were getting weird results of missing value sets when testing.

Reply:
done